src/app/app.module.ts

```diff
@@ -37,9 +45,18 @@ import 'rxjs/add/observable/of';
   imports: [
     RouterModule.forRoot(routes),
     StoreModule.provideStore(reducer),
+    TranslateModule.forRoot({
+      loader: {
+        provide: TranslateLoader,
+        useFactory: (createTranslateLoader),
+        deps: [HttpClient]
+      }
+    }),
+    HttpModule,
+    HttpClientModule,
     BrowserModule,
     FormsModule,
-    HttpModule,
+    HttpClientModule,
```

Comment:
@KnyazNiyaz do we have `HttpClientModule` twice ?

Reply:
@ashish173 Removed